Src/GoogleApis.Auth.DotNet4/OAuth2/ITokenAccess.cs

```diff
@@ -0,0 +1,40 @@
+﻿/*
+Copyright 2015 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using System.Threading;
+using System.Threading.Tasks;
+
+using Google.Apis.Auth.OAuth2.Responses;
+
+namespace Google.Apis.Auth.OAuth2
+{
+    /// <summary>
+    /// Provides members to retrieve auth token out-of-band.
+    /// This is necessary for workflows where you don't want to use 
+    /// <see cref="T:Google.Apis.Services.BaseClientService"/> to access the API.
+    /// If you don't know what this interface is for, chances are you don't need it.
```

Comment:
because gRPC :-) Explained in the OO design document.
Will fix the grammar in the comment.